.eslintrc

```diff
@@ -0,0 +1,22 @@
+{
+  "env": {
+    "node": true,
+    "es6": true
+  },
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "rules": {
+    "func-call-spacing": 2,
+    "guard-for-in": 2,
+    "no-debugger": 2,
+    "no-dupe-args": 2,
+    "no-dupe-keys": 2,
+    "no-duplicate-case": 2,
+    "no-multi-spaces": 2,
+    "no-self-compare": 2,
+    "no-unused-vars": 2,
```

Comment:
The `no-debugger` also caught something. The `no-multi-spaces` was also involved but as @lydell righteously mentioned it, we shouldn't interfere on the code style itself as this is where Prettier shines actually 👍.